### PR TITLE
OwnerType: support Bot from GitHub Apps

### LINKED ISSUE #18: Add support for “Bot” owner type
My GitHub App is listening for `InstallationRepositoriesEvent` and it also sometimes creates repositories itself and then receives an event about the repository it just created. Events for repositories created by the app have the following `sender`:

```
  "sender": {
    "login": "<application-slug>[bot]",
    <...>
    "type": "Bot",
    <...>
  }
```

Sender is being parsed as `HookUser`, whose `type` field is parsed as `OwnerType`, which currently supports only two values (`User` or `Organization`), so it fails with:

```
Error in $.sender.type: Unknown owner type: Bot
```

### PR DESCRIPTION
**Pull request by domenkozar**
Fixes #18

### CODE REVIEW

File: src/GitHub/Data/Webhooks/Payload.hs [hunk 1]
```diff
@@ -60,7 +60,7 @@ import           GHC.Generics             (Generic)
 -- Types lifted from the @github@ package.
 
 -- | Represents the owner of the repository.
```
Comment (roberth) [inline src/GitHub/Data/Webhooks/Payload.hs]:
```suggestion
-- | Represents the owner of a repository, pull request or similar.
--
-- A bot is a "special type of user which takes actions on behalf of GitHub Apps".
-- See also https://developer.github.com/v4/object/bot/
```

**Comment by donkeybonks** [comment]
Looks good, thank you.

**Comment by donkeybonks** [comment]
I will release this within the week to Hackage.

**Comment by donkeybonks** [comment]
Sorry for the delay in releasing this; I moved house this week.

**Comment by donkeybonks** [comment]
@domenkozar shall I add you to the authors list also?

**Comment by domenkozar** [comment]
As you'd like, I'm fine either way.

**Pull request merged**